packages/manager/apps/vrack-services/src/pages/subnets/Onboarding.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { ODS_BUTTON_SIZE, ODS_ICON_NAME } from '@ovhcloud/ods-components';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import { OnboardingLayout } from '@/components/layout-helpers/OnboardingLayout';
```

Comment:
No it's not possible because they are very different as you can see if you compare them

I would need to add options in the manager-components before doing so but the PR is about tracking here
Probably in the future